affordance_primitives/include/affordance_primitives/screw_model/affordance_utils.hpp

```diff
@@ -34,10 +34,10 @@
 #pragma once
 
 #include <optional>
+#include <sstream>
 
 #include <affordance_primitives/msg_types.hpp>
```

Comment:
maybe I'm missing something but... where do you have this header file?

Reply:
This is a C++ std library include, [see here](https://www.cplusplus.com/reference/sstream/stringstream/stringstream/)

Reply:
Upps sorry wrong line, I didn't mean the stream, I was asking for the <affordance_primitives/msg_types.hpp>

Reply:
Ah, that would be in [affordance_primitives/include/affordance_primitives/msg_types.hpp](https://github.com/UTNuclearRobotics/affordance_primitives/blob/noetic/affordance_primitives/include/affordance_primitives/msg_types.hpp)